live_pipeline/extract/test_extract.py

```diff
@@ -0,0 +1,44 @@
+import pytest
```

Comment:
Could add a docstring to the top of this file explaining what the file is used for an what it does